globals.go

```diff
@@ -525,6 +528,17 @@ const (
 	PatchNone            PatchPermission            = 0
 	LogsView             LogsPermission             = 10
 	LogsNone             LogsPermission             = 0
+
+	// Distro permissions.
+	PermissionDistros = "distro"
```

Comment:
I think it would be clearer to call these "distro settings" or something like that